src/Miunie.WindowsApp/Views/ImpersonationChatPage.xaml

```diff
@@ -34,9 +34,22 @@
                     <ListView ItemsSource="{Binding Messages}">
                         <ListView.ItemTemplate>
                             <DataTemplate x:DataType="core:MessageView">
-                                <StackPanel>
-                                    <TextBlock Text="{Binding AuthorName}"/>
-                                    <TextBlock Text="{Binding Content}"/>
+                                <StackPanel Orientation="Horizontal" Padding="10">
+                                    <Ellipse
+                                        Height="45"
+                                        Width="45"
+                                        VerticalAlignment="Top">
+                                        <Ellipse.Fill>
+                                            <ImageBrush ImageSource="{Binding AuthorAvatarUrl}"/>
+                                        </Ellipse.Fill>
+                                    </Ellipse>
```

Comment:
What about the [person picture control](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/person-picture)? Have you tried that one instead of an Ellipse?

Reply:
I'll see what I can do!